test_suite/error.slice_out_of_bounds.sugar.jsonnet

```diff
@@ -0,0 +1,3 @@
+local arr = std.range(0, 5);
+
+arr[0:10]
```

Comment:
Python slicing seems crazy complex: 

https://github.com/rfk/pypy/blob/master/pypy/objspace/std/listobject.py#L1362-L1382

I'll look into duplicating the behavior exactly into std.slice()